src/Message/DirectAuthorizeRequest.php

```diff
@@ -54,6 +54,8 @@ protected function getBaseAuthorizeData()
         $data['DeliveryPhone'] = $card->getShippingPhone();
         $data['CustomerEMail'] = $card->getEmail();
 
+        $data = array_merge($data, $this->getItemData());
```

Comment:
More consistent IMO would be for getItemData to return just the XML, and this line to add the element, rather than an array_merge. It's clearer what is going on that way:

    $data['BasketXML'] = $this->getItemData();

Question: how does Sage Pay cope with an empty XML basket? Would it be preferable to just to not send the basket element at all?

`unitTaxAmount` - 0.0 - is that a safe assumption? I realise the OmniPay `Item` class cannot easily be extended by a gateway to introduce in new data fields (last time I looked, at least). Zero is probably better than nothing for now.

Also "0.0" rather than "0.00"? The documentation does say it has to be formatted to 2dp (when appropriate for the currency), including trailing zeros. OmniPay has a built-in method for formatting currency amounts.